helper/helper_embark_general.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+# EMBArk - The firmware security scanning environment
+#
+# Copyright 2020-2022 Siemens Energy AG
+# Copyright 2020-2022 Siemens AG
+#
+# EMBArk comes with ABSOLUTELY NO WARRANTY.
+#
+# EMBArk is licensed under MIT
+#
+# Author(s): Benedikt Kuehne
+
+# Description: Helper functions
+
+docker_image_rm(){
+  # removes image by name and version
+  # $1 name
+  # $2 version
+  local IMAGE_NAME_="$1"
+  local IMAGE_VERSION_="$2"
+
+  if [[ $(docker image ls -q "$IMAGE_NAME_"":""$IMAGE_VERSION_" | wc -c ) -ne 0 ]] ; then
+    if [[ $(docker ps -a -q --filter "ancestor=""$IMAGE_NAME_"":""$IMAGE_VERSION_" | wc -c) -ne 0 ]]; then
+      local CONTAINERS_
+      mapfile -t CONTAINERS_ < <(docker ps -a -q --filter ancestor="$IMAGE_NAME_"":""$IMAGE_VERSION_" --format="{{.ID}}")
+      for CONTAINER_ID_ in "${CONTAINERS_[@]}" ; do
+        echo -e "$GREEN""$BOLD""Stopping ""$CONTAINER_ID_"" docker container""$NC"
+        docker stop "$CONTAINER_ID_"
+        echo -e "$GREEN""$BOLD""Remove ""$CONTAINER_ID_"" docker container""$NC"
+        docker container rm "$CONTAINER_ID_" -f
+      done
+    fi
+    echo -e "$GREEN$BOLD""Removing ""$IMAGE_NAME_"":""$IMAGE_VERSION_" "docker image""$NC\\n"
+    docker image rm "$IMAGE_NAME_"":""$IMAGE_VERSION_" -f
+  fi
+}
+
+docker_network_rm(){
+  # removes docker networks by name
+  local NET_ID
+  if docker network ls | grep -E "$1"; then
```

Comment:
please start setting up the vars in the beginning of the func. E.g.
local NAME_OF_VAR="${1:-}"